### ISSUE TITLE
Focal loss on ImageNet-LT

### ISSUE THREAD

**Issue by MiaoMiaoYoung** [KaihuaTang/Long-Tailed-Recognition.pytorch#32]
Thank you for your extraordinary work. Would you like to provide the config code (.yaml) of Focal Loss in Imagenet-LT？

**Comment by KaihuaTang** [comment]
I didn't write a separate config file for the focal loss. I just use the following implementation of focal loss [1] to replace the cross_entropy loss on the original baseline

[1] https://github.com/clcarwin/focal_loss_pytorch

**Comment by MiaoMiaoYoung** [comment]
Thanks a lot~!  (⁎˃ᴗ˂⁎)
